### DISCUSSION TITLE
Paid Consulting

### DISCUSSION THREAD

**Discussion by yujanshrestha** [Project-MONAI/MONAI#5190] [Q&A] [answered]
First of all I would like to thank the Monai community for such an amazing library. Before this I have never seen a library that solves so many recurring problems in medical imaging ML all under one roof. Keep up the great work! For example, the persistent cache data loader is a life saver compared to my rats nest of error prone makefiles generate a mass of intermediate files.

I am using Monai on several client projects and I am looking for members of the community that are open to doing some paid consulting or seeking employment. I could use some help with:

1. Formulating the best strategies for a given ML problem. Should I go just Unet? Unet++? MIx in transformers? Augment with stable diffusion? Use XYZ whizbang new library? Please tell me!
2. Getting me unstuck with difficult to diagnose bugs in my Monai application. A fresh set of eyes is very useful to overcome problems that could have me stuck for weeks.
3. Training a new model. I provide you data and funding to train a model.

Please contact here or via my email: yshrestha@innolitics.com

**Comment by pdogra89** [comment, marked as answer]
Hi Yujan

This is very exciting. There are companies like Kitware, Quantiphi who do paid consulting, development with MONAI. 
I would reach out to you on email and share how Nvidia team could potentially engage closely with innolitics to explore some options.

**Comment by yujanshrestha** [comment]
Sounds good! Please reach out to me on my email so we can get a conversation going. [yshrestha@innolitics.com](mailto:yshrestha@innolitics.com)

**Comment by andinet** [comment]
Hi Yujan

Please feel free to reach out if Kitware can help in your project. 

https://www.kitware.com/contact/project/

Here are some useful blogs that describe how Kitware is helping medical device companies and research organizations in AI/ML application development based on MONAI, ITK, 3D Slicer, VTK, and other open-source tools.

https://www.kitware.com/leveraging-monai-for-medical-ai/
https://www.kitware.com/kitware-medical-software-product-development/

Best,
Andinet

**Reply by yujanshrestha** [reply]
Thank you Andinet!
